src/Compilers/Core/CodeAnalysisTest/PEWriter/BlobTests.cs

```diff
@@ -13,6 +15,8 @@
 
 namespace Microsoft.CodeAnalysis.UnitTests.PEWriter
 {
+    using Roslyn.Reflection;
```

Comment:
I see -- didn't understand the type was duplicated, I thought these usings were just generated by some tool. Ignore the comments.